### PR TITLE
Document the sources for the sourceIPs audit log field

### PR DESCRIPTION
**Pull request by tallclair**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update the doc comment on the `sourceIPs` audit log field to indicate the sources included in the list. See https://github.com/kubernetes/kubernetes/pull/87167.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @mikedanese @deads2k 
/sig auth api-machinery
/priority important-longterm

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
Hi @tallclair 👋🏽 I'm from the Bug Triage team. We've crossed the Code Freeze for 1.20 release on 12th November. As this PR is tagged with 1.20, I'm sending a final reminder to either move the milestone to 1.21 or clear the milestone.

File: staging/src/k8s.io/apiserver/pkg/apis/audit/types.go [hunk 1]
```diff
@@ -98,6 +98,11 @@ type Event struct {
 	// +optional
 	ImpersonatedUser *authnv1.UserInfo
 	// Source IPs, from where the request originated and intermediate proxies.
+	// The source IPs are listed from (in order):
+	// 1. X-Forwarded-For request header IPs
+	// 2. X-Real-Ip header, if not present in the X-Forwarded-For list
+	// 3. The remote address for the connection, if it doesn't match the last IP in the list
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/apis/audit/types.go:104]: this is confusing to read. You want to say "in the list up to here, i.e. 1 and 2".

**Comment by kcmartin** [comment]
Hello! I am from the Bug Triage team! This PR has not been updated for a long time, so I'd like to check on the status. The code freeze is starting March 9th, 2021 (less than 1 week from now). We want to ensure that each PR has a chance to be merged before code freeze.
As the PR is tagged for 1.21, is it still planned for this release?

**Comment by erismaster** [comment]
Hi, Bug Triage Lead for 1.21 here, looks like this recently went stale, and we are past code freeze for 1.21. I am going to drop the milestone.

/milestone clear

File: staging/src/k8s.io/apiserver/pkg/apis/audit/types.go (hunk 1 above)
Comment (liggitt) [reply staging/src/k8s.io/apiserver/pkg/apis/audit/types.go:104]: looks like this comment is still outstanding

**Comment by tallclair** [comment]
I'll clean this up and try to get it merged soon.

**Comment by dims** [comment]
@tallclair one tiny nit left before we can land this. PTAL

**Comment by tallclair** [comment]
Sorry for letting this languish for so long. Updated.

**Comment by tallclair** [comment]
rebase due to removal of alpha & beta APIs.

**Comment by liggitt** [comment]
fixup the generated verify error, then lgtm

**Pull request merged**